live/tests/integration/components/qroc-solution-panel-test.js

```diff
@@ -0,0 +1,111 @@
+import { expect } from 'chai';
+import { describe, it, beforeEach } from 'mocha';
+import { setupComponentTest } from 'ember-mocha';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+
+const ANSWER_BLOCK = '.correction-qroc-box__answer';
+const ANSWER_INPUT = '.correction-qroc-box--answer__input';
+const SOLUTION_BLOCK = '.correction-qroc-box__solution';
+const SOLUTION_DISPLAY = '.correction-qroc-box__solution-text';
+
+const RIGHT_ANSWER_GREEN = 'rgb(19, 201, 160)';
+const NO_ANSWER_GREY = 'rgb(51, 51, 51)';
+
+describe('Integration | Component | qroc solution panel', function () {
+  setupComponentTest('qroc-solution-panel', {
+    integration: true
+  });
+
+  it('renders', function () {
+    this.render(hbs`{{qroc-solution-panel}}`);
+    expect(this.$()).to.have.length(1);
+  });
+
+  it('should disabled all inputs', function () {
+    // given
```

Comment:
il s'agit d'une clause `// when` et la ligne `const input = this.$('input');` doit plutôt être dans le `// then`